dags/process_elife_zip_dag.py

```diff
@@ -69,17 +79,38 @@ def extract_archived_files_to_bucket(**context):
         logger.info('ZIPPED FILES= %s', ZipFile(temp_file).namelist())
 
         # extract zip to cloud bucket
-        folder_name = zip_file_name.replace('.zip', '')
+        folder_name = zip_file_name.rstrip('.zip')
+        file_to_upload = None
+
         for zipped_file_path in ZipFile(temp_file).namelist():
-            if Path(zipped_file_path).suffix in ALLOWED_EXTENSIONS:
+
+            if is_image(zipped_file_path) and not is_allowed(zipped_file_path):
```

Comment:
time for some exploratory testing: take the TIFF files from the various articles https://github.com/elifesciences/elife-spectrum/tree/master/spectrum/templates and pass them through this conversion to see whether anything explodes. This would be a small sample, then we can take a larger one from the `prod-elife-published` bucket.

Reply:
In the assets folder there is `elife-36842-vor-r3.zip` which contains many tiff files. I think the tests you're thinking of we can do afterward as part of adding demo content to unstable. If there's a bug I'll raise a new issue.

Reply:
I guess https://github.com/libero/jats-ingester/pull/18/files#diff-5af835e7fbb86db53030f149d2de8d31R22 is the main coverage for this code then (separating code coverage from input coverage, which is the postponed part).